test/web-platform-tests/to-upstream/dom/attributes-are-nodes.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Attributes are Nodes but should not be accepted outside of the `attributes` NamedNodeMap</title>
```

Comment:
I’d like to move this file in a separate commit to preserve `git blame` history, like #2764.

Reply:
You could preserve the history here by making it a separate commit. We will then rebase while merging.

Reply:
I&nbsp;don’t&nbsp;think it&nbsp;really&nbsp;matters that&nbsp;much, unlike&nbsp;`XMLHttpRequest`.